docs/guides/token_guide.md

```diff
@@ -0,0 +1,30 @@
+# Chrysalis Migration
+
+With the advent of the Chrysalis migration, a number of aspects at IOTA are changing for the better. With easier ways to manage and secure your experience, IOTA fully intends to seamlessly integrate these innovations with no service interruptions to you.
+
+With Chrysalis, we make a clear cut from the current IOTA protocol, and start anew with a much better, and more mature network. The new network will support many new use cases and create a foundation for IOTA’s upcoming Coordicide.
+
+This includes one of the innovations that directly impacts one of the most crucial aspects of your experience: funds.
+
+Because of its complexity, the migration process for the next phase of Chrysalis consists of phases; we are currently in phase one where your funds will be migrated in one week and will be available on our new network upon launch.
+
+Do note that migrating is a continuous effort that can be done at any time but for your sake and expediency, we recommend migrating ahead of the launch to avoid any interruptions to your service.
+
+# Token Migration
+
+Below is an overview of how the migration will take place. For the more technical explanation of how the migration will work, see our **wallet** and **client libraries**.  And for a step by step overview, please see our **blog post**.
+
+
+
+## Migration Overview
+
+- You enter your seed in Firefly.
+- Firefly creates you a new seed and generates an EdDSA address for the new network.
+- Firefly sends your funds to a predetermined migration address on the old network.
+- Your funds become available on the network, in the EdDSA address Firefly created for you. 
+- If you migrate before Chrysalis launch, your funds become available at Chrysalis launch. 
+- If you migrate after Chrysalis launch, your funds will become available shortly after you migrate.
+- The process will be the same for Ledger Nano users. 
```

Comment:
Maybe add a comment, that Ledger will not be supported before the Chrysalis launch, but later